drag_and_drop/drag_and_drop.py

```diff
@@ -353,7 +352,7 @@ def _inner_content(self, tag):
         """
         inner_content = None
         if tag is not None:
-            inner_content = ''.join(ET.tostring(e) for e in tag)
+            inner_content = ''.join(ET.tostring(e).decode('utf-8') for e in tag)
```

Comment:
@attiyaIshaque try if `ET.tostring(e, encoding='unicode')` works instead of using decode()

Reply:
I have tried it but it shows me error.